### DISCUSSION TITLE
Prisma not finding Database URL env variable

### DISCUSSION THREAD

**Discussion by ThomasAlban** [prisma/prisma#21544] [Q&A]
### Question

Hi,
When I run `npx prisma db push` I get this error:
```
Error: Prisma schema validation - (get-config wasm)
Error code: P1012
error: Environment variable not found: DATABASE_URL.
  -->  schema.prisma:10
   |
9 |     provider = "postgresql"
10 |     url      = env("DATABASE_URL")
   |
 
Validation Error Count: 1
[Context: getConfig]
```
I'm using this with SvelteKit, and have already run `npx prisma generate` and `npm run dev` to generate types (as suggested online), which both seemed to work fine. However `npx prisma db push` still gives me this error.
I have also checked that I have a `.env` file in the root of my project which contains the database url and I've checked all the syntax.
Any help appreciated!

**Comment by ludralph** [comment]
Hi @ThomasAlban 👋 

Thank you for raising this question.

Can you check that you are running the command in the root project and not in another directory also is it possible to share a minimal reproduction of this in a repository so I can investigate this further?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
